service/service.go

```diff
@@ -100,35 +14,103 @@ func (s *Service) GetDependency(dependencyKey string) (*Dependency, error) {
 	return nil, fmt.Errorf("dependency %s do not exist", dependencyKey)
 }
 
-// ValidateConfigurationEnv checks presence of env variables in mesg.yml under env section.
-func (s *Service) ValidateConfigurationEnv(env map[string]string) error {
-	var nonDefined []string
-	for key := range env {
-		exists := false
-		// check if "key=" exists in configuration
-		for _, env := range s.Configuration.Env {
-			if strings.HasPrefix(env, key+"=") {
-				exists = true
-			}
+// GetTask returns task taskKey of service.
+func (s *Service) GetTask(taskKey string) (*Task, error) {
+	for _, task := range s.Tasks {
+		if task.Key == taskKey {
+			return task, nil
 		}
-		if !exists {
-			nonDefined = append(nonDefined, key)
+	}
+	return nil, &TaskNotFoundError{
```

Comment:
Remove TaskNotFoundError error

Reply:
why?

Reply:
Because when you don't need error types, returning `fmt.Errorf` is enough. In general, you don't create types for error when you don't check them

Reply:
I don't want to change any behavior in this PR, this might be done in other PR if it's needed